packages/cli/src/modules/chat-hub/chat-hub.service.ts

```diff
@@ -836,7 +879,6 @@ export class ChatHubService {
 			},
 		};
 
-		const startNodes: StartNodeData[] = [{ name: 'Restore Chat Memory', sourceData: null }];
```

Comment:
With `startNodes`, the engine performs partial execution and doesn't run all nodes.